javascript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+let count = 4;
+
+const countdown = function startCountdown() {
+  const $countdownDisplay = document.querySelector('.countdown-display');
+  count--;
+  $countdownDisplay.textContent = count;
+  if (count < 1) {
+    $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(countdown);
```

Comment:
You're passing in the wrong argument here. The argument should be an id that gets returned by the `setInterval` function